packages/cra-template-safe-app/template/src/App.tsx

```diff
@@ -1,32 +1,33 @@
-import React, { useCallback, useState } from 'react';
-import styled from 'styled-components';
-import { Button, Loader, Title } from '@gnosis.pm/safe-react-components';
-import { useSafeAppsSDK } from '@gnosis.pm/safe-apps-react-sdk';
+import React, { useCallback } from "react";
+import styled from "styled-components";
+import { Button, Title } from "@gnosis.pm/safe-react-components";
+import { useSafeAppsSDK } from "@gnosis.pm/safe-apps-react-sdk";
 
-const Container = styled.form`
-  margin-bottom: 2rem;
+const Container = styled.div`
+  padding: 1rem;
   width: 100%;
-  max-width: 480px;
-
-  display: grid;
-  grid-template-columns: 1fr;
-  grid-column-gap: 1rem;
-  grid-row-gap: 1rem;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
 `;
 
-const App: React.FC = () => {
+const Link = styled.a`
+  margin-top: 8px;
+`
+
+const SafeApp = (): React.ReactElement => {
   const { sdk, safe } = useSafeAppsSDK();
-  const [submitting, setSubmitting] = useState(false);
 
   const submitTx = useCallback(async () => {
-    setSubmitting(true);
     try {
       const { safeTxHash } = await sdk.txs.send({
         txs: [
           {
             to: safe.safeAddress,
-            value: '0',
-            data: '0x',
+            value: "0",
```

Comment:
Didn't see this before. Is this project now using double quotes?

Reply:
oops, prettier config was invalid. fixed